src/annotator/util/shadow-root.js

```diff
@@ -29,8 +29,8 @@ function loadStyles(shadowRoot) {
  * In browsers that support it, shadow DOM is used to isolate annotator UI
  * components from the host page's styles.
  *
- * @param {HTMLElement} - Container element to render the UI into
- * @return {Element} -
+ * @param {HTMLElement} container - Container element to render the UI into
+ * @return {HTMLElement|ShadowRoot} -
```

Comment:
Curious how you figured out that you needed to provide `ShadowRoot` as an alternative here...

Reply:
The error message from the TS compiler told me.